src/Components/Admin/navbar.js

```diff
@@ -49,39 +54,56 @@ export default function Navbar({ home }) {
                             <HomeIcon />
                         </IconButton>
                     )}
-                    <div className={classes.profileEnd}>
-                        <IconButton
-                            aria-label="account of current user"
-                            aria-controls="menu-appbar"
-                            aria-haspopup="true"
-                            onClick={handleMenu}
-                            color="inherit"
-                        >
-                            <AccountCircle />
-                        </IconButton>
-                        <Menu
-                            id="menu-appbar"
-                            anchorEl={anchorEl}
-                            anchorOrigin={{
-                                vertical: "bottom",
-                                horizontal: "center",
-                            }}
-                            keepMounted
-                            transformOrigin={{
-                                vertical: "top",
-                                horizontal: "right",
-                            }}
-                            open={open}
-                            onClose={handleClose}
-                            className={classes.menu}
-                        >
-                            <MenuItem onClick={handleClose}>
-                                Documentation
-                            </MenuItem>
-                            <MenuItem onClick={handleClose}>Settings</MenuItem>
-                            <MenuItem onClick={handleClose}>Logout</MenuItem>
-                        </Menu>
-                    </div>
+                    {user && (
```

Comment:
I assume this would prevent the menu from even showing up if you're logged out?

Reply:
The menu bar loads but the right most icon doesn't load so you cannot instigate menu.

Reason for this -- if we have a name or logo present in the navbar, we would probably still want that to load on login page. But no functionality (home, that menu dropdown) should load.

Reply:
@andstun Perfect example of conditional rendering! You will need it in an upcoming ticket